Source/Mosa.DeviceSystem/Bitmap.cs

```diff
@@ -9,9 +9,9 @@ public static Image CreateImage(byte[] data)
 			if (data[0] != (byte)'B' || data[1] != (byte)'M')
 				return null;
 
-			var bpp = data[0x1C] / 8;
+			var bpp = (uint)(data[0x1C] / 8);
 
-			if (!(bpp == 3 || bpp == 4))
+			if (bpp != 3 || bpp != 4)
```

Comment:
The logic is incorrect here, need `&&` because you are wanting to do is if bpp is neither 3 nor 4 then return null

Reply:
Ah that's fair, didn't quite take a look at that.